app/models/lookup_keys/lookup_key.rb

```diff
@@ -103,7 +103,8 @@ def supports_uniq?
   end
 
   def to_param
-    Parameterizable.parameterize("#{id}-#{key}")
+    search_key = key.tr(' ','_') unless key.nil?
```

Comment:
yep, sorry it was about the migration. I'll remove this since the migration will take care of it.